source/configuration_manual/mail_location/dbox/index.rst

```diff
@@ -0,0 +1,134 @@
+.. _dbox_settings:
+
+==================
+Dbox Configuration
+==================
+
+See :ref:`dbox <dbox_mbox_format>` for a technical description of Dovecot's
+dbox mailbox format.
+
+NOTE: **You must not lose the dbox index files, as they can't be regenerated
+without data loss**.
+
+Mail Location
+^^^^^^^^^^^^^
+
+To use **single-dbox**, use the tag ``sdbox`` in the
+:ref:`mail location <mail_location_settings>`:
+
+.. code-block:: none
+
+  # single-dbox
+  mail_location = sdbox:~/dbox
+
+For backwards compatibility, ``dbox`` is an alias to ``sdbox`` in the mail
+location. (This usage is deprecated.)
+
+To use **multi-dbox**, use the tag ``mdbox`` in the
+:ref:`mail location <mail_location_settings>`:
+
+.. code-block:: none
+
+  # multi-dbox
+  mail_location = mdbox:~/mdbox
+
+.. _dbox_settings_alt_storage:
+
+Alternate Storage
+^^^^^^^^^^^^^^^^^
+
+See :ref:`dbox alternate storage <dbox_mbox_format_alt_storage>` for further
+information.
+
+To specify an alternate storage area, use the ``ALT`` parameter in the mail
+location. For example:
+
+.. code-block:: none
+
+  mail_location = mdbox:/var/vmail/%d/%n:ALT=/altstorage/vmail/%d/%n
+
+will make Dovecot look for message data first under ``/var/vmail/%d/%n``
+("primary storage"), and if it is not found there it will look under
+``/altstorage/vmail/%d/%n`` ("alternate storage") instead. There's no problem
+having the same (identical) file in both storages.
+
+Keep the unmounted ``/altstorage`` directory permissions such that Dovecot
+mail processes can't create directories under it (e.g. ``root:root 0755``).
+This way if the alt storage isn't mounted for some reason, Dovecot won't
+think that all the messages in alt storage were deleted and lose their flags.
+
+.. _dbox_settings_mailbox_directory_name:
+
+Mailbox Directory Name
+^^^^^^^^^^^^^^^^^^^^^^
+
```

Comment:
This setting should really be removed and I think we should discourage people from using it.